src/shared/database/schema.ts

```diff
@@ -68,6 +73,22 @@ export interface DexExAggregateSummary {
   usdc_price: number | null;
 }
 
+export interface DexExBatchSwapTraces {
+  amount_hops: ArrayType<Numeric>;
+  asset_end: Buffer;
+  asset_hops: ArrayType<Buffer>;
+  asset_start: Buffer;
+  batch_input: Numeric;
+  batch_output: Numeric;
+  height: number;
+  input: Numeric;
+  output: Numeric;
+  position_id_hops: ArrayType<Buffer>;
```

Comment:
This is unimplemented for now, it requires protocol changes. Will x-ref the issue.